Makefile

```diff
@@ -172,6 +172,21 @@ install-python-linter:
 python-lint:
 	scripts/lint.sh
 
+## install-python-formatter: Install python formatter
+install-python-formatter:
+	pip install black==20.8b1
+
+## python-format: Format python packages
+python-format:
+	black packages/sdk
+	black packages/cli
+	black packages/robot
+
+## configure-git: Configure repo git config
+configure-git:
```

Comment:
```suggestion
## config-git-blame-ignore: Configure repo git config to ignore revisions specified into file
config-git-blame-ignore:
```

Reply:
This command will configure not only git-blame but other git configuration as well, for example git hooks etc

Reply:
OK